src/sentry/identity/gitlab/provider.py

```diff
@@ -39,8 +43,18 @@ def get_user_info(access_token, installation_data):
         },
         verify=installation_data['verify_ssl']
     )
-
-    resp.raise_for_status()
+    try:
+        # HELP(LB): is this reasonable?!?!?
+        resp.raise_for_status()
+    except Exception as e:
+        logger.info('gitlab.identity.get-user-info-failure',
+                    extra={
+                        'url': installation_data['url'],
+                        'verify_ssl': installation_data['verify_ssl'],
+                        'client_id': installation_data['client_id']
```

Comment:
Do you want the the response body or status code? Both might help in understanding more about what is failing. If you didn't want to use try/accept you could do something like:

```python
if not resp.ok:
  logger.info(...)
  resp.raise_for_status()
```